src/__tests__/use-document/use-document.test.tsx

```diff
@@ -91,4 +103,52 @@ describe('_document support', () => {
       });
     }
   );
+
+  describe('next/head', () => {
+    describe('first render', () => {
+      it('merges _document and page head elements', async () => {
```

Comment:
Do we have a test where `_document` + `_app` + `page` Head element?

Reply:
I don't think so, but `next/head` test group in `use-document` might definitely be the right candidate. All that's missing is using `next/head` in `src/__tests__/use-document/__fixtures__/custom-document-with-gip/pages/_app.js`.

This would also make `next/head` tests more complete 👍